submissions/kirill8210/popup-task/index.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>html_css_popup</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+<div class="container">
+    <header class="header">
+        <a href="#"><img src="img/favicon.png" alt="favicon"></a>
+        <nav>
+            <ul class="menu">
+                <li class="menu_li"><a href="#">Mail</a></li>
+                <li class="menu_li"><a href="#">Images</a></li>
+                <li class="menu_li">
+                    <input id="checkbox" class="checkbox" type="checkbox">
+                    <label for="checkbox">
+                        <img src="img/popup-button.png" alt="">
+                    </label>
+                    <ul class="popup">
+                        <li class="popup_li"><a href="#"><img src="img/calendar.png" alt="calendar"><p>Calendar</p></a></li>
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

Reply:
ok